keps/976-plain-pods/README.md

```diff
@@ -0,0 +1,669 @@
+# KEP-976: Plain Pods
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+    - [Story 3](#story-3)
+  - [Story 4](#story-4)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+    - [Skipping Pods belonging to queued objects](#skipping-pods-belonging-to-queued-objects)
+    - [Pods replaced on failure](#pods-replaced-on-failure)
+  - [Risks and Mitigations](#risks-and-mitigations)
+    - [Increased memory usage](#increased-memory-usage)
+- [Design Details](#design-details)
+  - [Gating Pod Scheduling](#gating-pod-scheduling)
+    - [Pods subject to queueing](#pods-subject-to-queueing)
+  - [Constructing Workload objects](#constructing-workload-objects)
+    - [Single Pods](#single-pods)
+    - [Groups of Pods with the same shape](#groups-of-pods-with-the-same-shape)
+    - [Groups of pods with multiple shapes or roles](#groups-of-pods-with-multiple-shapes-or-roles)
+    - [Groups of pods where driver generates workers](#groups-of-pods-where-driver-generates-workers)
+  - [Tracking admitted and finished Pods](#tracking-admitted-and-finished-pods)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+  - [Users create a Workload object beforehand](#users-create-a-workload-object-beforehand)
+<!-- /toc -->
+
+## Summary
+
+Some batch applications create plain Pods directly, as opposed to managing the Pods through the Job
+API or a CRD that supports [suspend](https://kubernetes.io/docs/concepts/workloads/controllers/job/#suspending-a-job) semantics.
+This KEP proposes mechanisms to queue plain Pods through Kueue, individually or in groups,
+leveraging [pod scheduling gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/).
+
+## Motivation
+
+Some batch systems or AI/ML frameworks create plain Pods to represent jobs or tasks of a job.
```

Comment:
```suggestion
Some batch systems or AI/ML frameworks create plain Pods to represent Jobs or tasks of a Job.
```
This is a huge nit.  I noticed that the docs team tends to prefer caps for K8 resources and you capitalize Pods but not jobs throughout.

Reply:
in this case, it doesn't refer to k8s resources, but the generic idea of job

Reply:
Sounds good.